### DISCUSSION TITLE
Show 'files' total in sidebar (just like note totals)

### DISCUSSION THREAD

**Discussion by OddHouseGames** [standardnotes/forum#2841] [Suggestions]
The 'Files' item in the sidebar should show a total for how many files are currently in existence.
<img width="298" alt="Screen Shot 2022-07-27 at 3 14 37 AM" src="https://user-images.githubusercontent.com/62911518/181185659-4f24f187-74a9-4a70-8f52-8675eb200387.png">

**Comment by myreli** [comment]
This is now implemented: 
<img width="202" alt="image" src="https://user-images.githubusercontent.com/17554234/221363595-ceb8bfc7-6bf2-4c29-a18e-9334bb9e3921.png">
